spec/locale-sensitive-functions.html

```diff
@@ -65,27 +65,23 @@ <h1>String.prototype.toLocaleLowerCase ( [ _locales_ ] )</h1>
         1. Else,
           1. Let _requestedLocale_ be DefaultLocale().
         1. Let _noExtensionsLocale_ be the String value that is _requestedLocale_ with all Unicode locale extension sequences (<emu-xref href="#sec-unicode-locale-extension-sequences"></emu-xref>) removed.
-        1. Let _availableLocales_ be a List with the language tags of the languages for which the Unicode character database contains language sensitive case mappings.
+        1. Let _availableLocales_ be a List with language tags that includes the languages for which the Unicode Character Database contains language sensitive case mappings. Implementations may add additional language tags if they support custom tailoring.
         1. Let _locale_ be BestAvailableLocale(_availableLocales_, _noExtensionsLocale_).
         1. If _locale_ is *undefined*, let _locale_ be `"und"`.
         1. Let _cpList_ be a List containing in order the code points of _S_ as defined in ES2020, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, starting at the first element of _S_.
-        1. For each code point _c_ in _cpList_, if the Unicode Character Database provides a lower case equivalent of _c_ that is either language insensitive or for the language _locale_, replace _c_ in _cpList_ with that/those equivalent code point(s).
+        1. For each code point _c_ in _cpList_, replace _c_ with a lower case equivalent of _c_ that is either language insensitive or defined according to the _locale_ specific code point mapping. Implementations must always be consistent in the case mapping they use for a specified locale.
```

Comment:
This "consistent" wording is interesting, but it could be a little confusing. Could you use phrasing that indicates, the output of this method is a function of the locale data and its inputs? E.g., "In a particular implementation, the transformation of _cpList_ always yields the same result, for any fixed _cpList_ and locale." (Maybe you have ideas for making that clearer.)

Note, the "for each code point" wording here is misleading; I fixed it in ECMA-262 in https://github.com/tc39/ecma262/pull/932/files , and we should probably apply the same fix in ECMA-402. (Could be a separate PR, or the same one, as you're already touching this line.)

Reply:
I like your replacement for "consistent" - I'll add it.

Do you think code points is used clearly in many places in this spec? If so can you file and issue? :)

Reply:
I am not aware of other instances of this error outside of case mapping.

Reply:
I think I didn't understand what was misleading about the "for each code point" until now, but I see -- the mapping isn't always directly from on code point to another.